consensus/ConsensusTableUpdater.py

```diff
@@ -0,0 +1,338 @@
+        :param msg: message showing when busy deleting table
+        :param table: fully qualified name of table to delete data from
+        :return: None
+        """
+        with yaspin(text=msg, color='green') as spinner:
+            try:
+                self.molgenis_server.delete(table)
+                spinner.ok('✔')
+            except Exception as e:
+                spinner.fail('💥')
+                cprint("Error while deleting data from [{}]: {}".format(table, e), 'red', attrs=['bold'],
+                       file=sys.stderr)
+
+    def _upload_file(self, file_name):
+        """
+        Uploads data from csv file to molgenis table
+        :param file_name: name of the file to upload (should be *fully qualified name here*.csv)
+        :return: True if Finishes
+        :raises ImportError if import failed
+        """
+        response = self.molgenis_server.upload_zip(file_name).split('/')
+        run_entity_type = response[-2]
+        run_id = response[-1]
+        status_info = self.molgenis_server.get_by_id(run_entity_type, run_id)
+
+        while status_info['status'] == 'RUNNING':
+            status_info = self.molgenis_server.get_by_id(run_entity_type, run_id)
+
+        if status_info['status'] == 'FINISHED':
+            return True
+        else:
+            raise ImportError(status_info['message'])
+
+    def upload_comments(self, consensus_data):
+        """
+        Creates csv for comments and uploads it
+        :param consensus_data: dictionary with variant ids (chr_pos_ref_alt_gene) as key
+        :return: None
+        """
+        comments_file_name = self.comments_table + '.csv'
+
+        print('\nUploading comments')
+
+        # Comments have as id the id of the variant and by default the value is "-"
+        comments = ['"{}","-"\n'.format(variant) for variant in consensus_data]
+
+        progress = 0
+
+        # Reserve 100 for the last step (uploading the file to molgenis)
+        progress_bar = progressbar.ProgressBar(max_value=len(comments) + 100)
+
+        # Open comments file and add header
+        comments_file = open(comments_file_name, 'w')
+        comments_file.write('"id","comments"\n')
+
+        # Add comments to comments file
+        for comment in comments:
+            comments_file.write(comment)
+            progress += 1
+            progress_bar.update(progress)
+
+        comments_file.close()
+
+        done_uploading = self._upload_file(comments_file_name)
+
+        if done_uploading:
+            progress += 100
+            progress_bar.update(progress)
+
+        progress_bar.finish()
+
+    def upload_consensus(self, file_name, consensus_len):
+        """
+        Upload consensus table
+        :param file_name: name of the file to upload
+        :param consensus_len: the number of all unique variants in the consensus table
+        :return: None
+        """
+        with yaspin(text='Uploading consensus table', color='green') as spinner:
```

Comment:
another synchronous upload (what's the diff between upload_file and upload_zip?)